docker.local/config/sc.yaml

```diff
@@ -218,7 +219,7 @@ smart_contracts:
       curator_transfer_allocation: 100
       challenge_request: 100
       challenge_response: 1600
-      generate_challenges: 9999 #exceeds limit, might be switched off
+      generate_challenges: 0
```

Comment:
Generate challenge should have been optimized, so perhaps we ca use 100 for it now.